VideoLocker/src/main/java/org/edx/mobile/util/Config.java

```diff
@@ -60,6 +67,40 @@
     private static final String SERVER_SIDE_CHANGED_THREAD = "SERVER_SIDE_CHANGED_THREAD";
     private static final String END_TO_END_TEST = "END_TO_END_TEST";
 
+    public static class AppUpdateUrisConfig {
+        @Nullable
+        @SerializedName("NATIVE")
+        private String nativeUri;
+
+        @Nullable
+        @SerializedName("WEB")
+        private String webUri;
+
+        public AppUpdateUrisConfig() {}
+
+        public AppUpdateUrisConfig(@NonNull final String nativeUri, @NonNull final String webUri) {
+            this.nativeUri = nativeUri;
+            this.webUri = webUri;
+        }
+
+        @Nullable
+        public String getNativeUri() {
+            return formatApplicationId(nativeUri);
+        }
+
+        @Nullable
+        public String getWebUri() {
+            return formatApplicationId(webUri);
+        }
+
+        private static String formatApplicationId(@NonNull final String uri) {
```

Comment:
A javadoc here would be helpful explaining the format of the Uri and what this function does.